### PR TITLE
v3.3: New param in CustoemrAPI::GetPredictedAgent.

### PR DESCRIPTION
**Pull request by gluszczyk**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7915-predicted-agent)
- [Jira](https://livechatinc.atlassian.net/browse/API-7659)

### Content

- New content
- Proofreading/content fixes

### CODE REVIEW

**Review by lpmichalik — APPROVED** [review]

File: content/messaging/customer-chat-api/changelog/index.mdx [hunk 1]
```diff
@@ -48,6 +48,12 @@ This document is the record of all the changes in the **Customer Chat API**, Web
   - the `group_id` parameter was renamed to `id`.
 - In the [**List License Properties**](/messaging/customer-chat-api/v3.3/#list-license-properties) method, the `license_id` parameter was removed.
 
+### Other
+
+- The **Get Predicted Agent** ([Web](/messaging/customer-chat-api/v3.3/#get_predicted_agent) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#get_predicted_agent)) response has a new format:
+  - previous fields are now nested together as the `agent` field,
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/changelog/index.mdx:54]:
```suggestion
  - The fields are now nested together as the `agent` field.
```

Comment (ale-kacperczyk) [reply content/messaging/customer-chat-api/changelog/index.mdx:54]: "Previous fields" suggests that there are new fields that replaced the old ones, while, in fact, they still exist, but have a different form.

File: content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx [hunk 2]
```diff
@@ -2224,6 +2224,12 @@ Gets the predicted Agent - the one the Customer will chat with when the chat sta
 | **Web API equivalent** | [`get_predicted_agent`](../#get-predicted-agent) |
 | **Push message**       | -                                                |
 
+#### Response
+
+| Field   | Data type | Notes                         |
+| ------- | --------- | ----------------------------- |
+| `queue` | `bool`    | Chat will be queued if `true` |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx:2231]:
```suggestion
| `queue` | `bool`    | If `true`, the chat will be queued. |
```

Comment (ale-kacperczyk) [reply content/messaging/customer-chat-api/v3.3/rtm-reference/index.mdx:2231]: Idk, it just sounds better in this order.

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**